packages/editor-ui/src/components/RunData.vue

```diff
@@ -253,11 +253,14 @@
 				<NodeErrorView :error="subworkflowExecutionError" :class="$style.errorDisplay" />
 			</div>
 
-			<div v-else-if="!hasNodeRun" :class="$style.center">
+			<div v-else-if="!hasNodeRun && !(isInputSchemaView && node?.disabled)" :class="$style.center">
 				<slot name="node-not-run"></slot>
 			</div>
 
-			<div v-else-if="paneType === 'input' && node?.disabled" :class="$style.center">
+			<div
+				v-else-if="paneType === 'input' && node?.disabled && !isInputSchemaView"
```

Comment:
Nitpick : could you please rearrange the order of these conditions 
```suggestion
				v-else-if="paneType === 'input' && !isInputSchemaView && node?.disabled"
```